packages/utils/test/composeAbortSignals.test.ts

```diff
@@ -0,0 +1,109 @@
+            controllers[4].abort()
+            expect(composedSignal.aborted).toBeTrue()
+        })
+
+        it('event "abort" is emitted when a signal aborts', () => {
+            const listener = jest.fn()
+            composedSignal.addEventListener('abort', listener)
+
+            expect(listener).toHaveBeenCalledTimes(0)
+            controllers[6].abort()
+            expect(listener).toHaveBeenCalledTimes(1)
+            expect(listener.mock.calls[0][0].type).toEqual('abort')
+        })
+
+        it('event "abort" is emitted max once', () => {
+            const listener = jest.fn()
+            composedSignal.addEventListener('abort', listener)
+            controllers[3].abort()
+            controllers[6].abort()
+            expect(listener).toHaveBeenCalledTimes(1)
+        })
+
+        it('onabort if set is invoked when a signal aborts', () => {
+            const listener = jest.fn()
+            ;(composedSignal as any).onabort = listener
+
+            expect(listener).toHaveBeenCalledTimes(0)
+            controllers[6].abort()
+            expect(listener).toHaveBeenCalledTimes(1)
+            expect(listener.mock.calls[0][0].type).toEqual('abort')
+        })
+
+        it('onabort is invoked max once', () => {
+            const listener = jest.fn()
+            ;(composedSignal as any).onabort = listener
+            controllers[3].abort()
+            controllers[6].abort()
+            expect(listener).toHaveBeenCalledTimes(1)
+        })
+    })
+
+    describe('given at least one pre-aborted signal', () => {
+        beforeEach(() => {
+            controllers = range(10).map(() => new AbortController())
+            controllers[8].abort()
+            composedSignal = composeAbortSignals(...controllers.map((c) => c.signal))
+        })
+
+        it('initial state is aborted=true', () => {
+            expect(composedSignal.aborted).toBeTrue()
+        })
+
+        it('another signal aborting has no impact on state', () => {
+            controllers[6].abort()
+            expect(composedSignal.aborted).toBeTrue()
+        })
+
+        it('another signal aborting does not emit "abort" event', () => {
+            const listener = jest.fn()
+            composedSignal.addEventListener('abort', listener)
+
+            expect(listener).toHaveBeenCalledTimes(0)
+            controllers[0].abort()
+            expect(listener).toHaveBeenCalledTimes(0)
+        })
+
+        it('another signal aborting does not cause onabort to be invoked', () => {
+            const listener = jest.fn()
+            ;(composedSignal as any).onabort = listener
+
+            expect(listener).toHaveBeenCalledTimes(0)
+            controllers[0].abort()
+            expect(listener).toHaveBeenCalledTimes(0)
+        })
+    })
+
+    it('throws error if composing empty list', () => {
+        expect(() => {
+            composeAbortSignals()
+        }).toThrowError(new Error('must provide at least one AbortSignal'))
```

Comment:
```suggestion
        }).toThrow('must provide at least one AbortSignal')
```

As the `jest` assertion doesn't check the error class anyway.

(same change for line `107`)

Reply:
Fixed 👍